Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -96,6 +96,35 @@ final class CacheAdvanceTests: XCTestCase {
         XCTAssertEqual(messages, [])
     }
 
+  func test_messages_throwsFileCorruptedWhenOffsetInFileAtEndOfNewsetMessageOutOfSync() throws {
```

Comment:
Nit: the indentation on this method is off.

Reply:
I'll update this in my next PR

Reply:
https://github.com/dfed/CacheAdvance/pull/58